tsp-core/src/main/java/pl/edu/agh/iet/tsp/core/config/DatabaseConfig.java

```diff
@@ -0,0 +1,51 @@
+package pl.edu.agh.iet.tsp.core.config;
```

Comment:
brzmi sensownie. jedyne co to trzeba by zmienic nazwe tsp-core bo zupelnie nie pasuje. core to jest raczej cos od czego zalezy wszystko inne, a nie na odwrot. mozna by to nazwac np controller. sproboje to wszystko skonczyc do soboty do wieczora.